### DISCUSSION TITLE
storyStoreV7 Diagnose invalid story

### DISCUSSION THREAD

**Discussion by cgatian** [storybookjs/storybook#16973] [Help]
I'm attempting to use the `storyStoreV7`. When I launch my storybook I receive the following error on any story.

![image](https://user-images.githubusercontent.com/1752170/145504256-cc174f48-85d2-4c77-a623-43f36f0fb794.png)

I found that if I select a small subset of stories from my project, it actually works 🎊 .
My guess is somewhere within my project there is a story with an invalid format. Is there a way for me to determine which story or stories may not have default properties set?

Edit:
Manually went through the stories and all seem to be exporting `title` and `component`. Continuing to investigate.

**Comment by shilman** [comment]
do you have a reproduction you can share? i can look into providing a better error. thanks!

**Reply by cgatian** [reply]
Yeah sorry, unclear what actually is causing this. I suspect it has something to do with the main.js>stories[] provided in the config. When I manually specify stories to use, instead of utilizing a globbing pattern it seems to work.

**Reply by cgatian** [reply]
Yeah, I think it's definitely related to something astray with the globs.
https://github.com/storybookjs/storybook/issues/16924
Closing this and will use issue to report any findings.
